# CapsidDB<sup>®</sup> [![Apache License](https://img.shields.io/badge/license-Apache-blue.svg)](https://github.com/CapsidDB/CapsidDB/blob/master/LICENSE)
<i><b>"</b>The truth is rarely pure and never simple.<b>"</b></i> - <b>Oscar Wilde</b>.

<b>CapsidDB<sup>®</sup>:</b>&nbsp;A Capsid proteins and their associated functional genes database.

## Capsid
A <b>capsid</b> is the protein shell of a virus. Virus particles contain the viral genome packaged in a <i>protein coat</i> called the <b>capsid</b>. For some viruses, the capsid is surrounded by lipid bilayer that contains viral proteins, usually including the proteins that enable the virus to bind to the host cells. This lipid and protein structure is called the virus envelope, and is derived from the host cell membranes. The capsid and envelope play many roles in viral infection, including virus attachment to cells, entry into cells, release of the capsid contents into the cells, and packaging of newly formed viral particles. The <i>capsid</i> and <i>envelope</i> are also responsible for transfer of the viral genetic material from one cell to another. These structures also determine the stability characteristics of the virus particle, such as resistance to chemical or physical inactivation.

### Structural Facts
A well known fact about <b>capsid</b> is, it encloses the genetic material of the virus and it consists of several oligomeric structural subunits made of protein called <b>protomers</b>. The observable 3-dimensional morphological subunits, which may or may not correspond to individual proteins, are called <b>capsomeres</b>.

The <b>capsomere</b> is a subunit of the capsid, an outer covering of protein that protects the genetic material of a virus. Capsomeres self-assemble to form the capsid. It's subunits called protomers aggregate to form capsomeres. Various arrangements of capsomeres are: (1) Icosahedral, (2) Helical, and (3) Complex.

### Genomics
The composition and structure of viral genomes is more varied than bacterial, plant, or animal kingdoms. The viral genomes may be <b>single-stranded</b> or <b>double-stranded</b>, <b>linear</b> or <b>circular</b>, and in <b>single-segmented</b> or <b>multi-segmented</b> configuration. There are 2,418 reference viral genomes in the current GenBank collection (up to 2012-4-6).

<b>Key features:</b>
- Viral genetic material is packaged inside protein structures called capsids.
- Viruses are divided into two groups: enveloped viruses are surrounded by an outer lipid membrane; non-enveloped viruses lack this membrane.
- Where present, the envelope contains the viral proteins, which mediate binding to host cells. Where no envelope is present, this function is carried out by the outer capsid proteins. In general, non-enveloped viruses are more stable and can survive much longer in the environment.
- Capsids and envelopes determine the method of viral entry into and exit from host cells.

<b>Note:</b> Viruses are known to be the most abundant organisms on earth and their associated expressed viral diseases have an enormous impact on human health worldwide.

### Proteomics
Viral genomes were chronologically <i>the first to be sequenced</i> and the corresponding proteomes are only now elucidated, with some surprising results.

Viruses replicate and proliferate in host cells while continuously adjusting to and modulating the host environment. They encode a wide spectrum of multifunctional proteins, which interplay with and modify proteins in host cells. However, the corresponding viral proteomes, that expressed through genes; the alterations of host proteomes upon viral infection, and the dynamic nature of proteins, such as post-translational modifications, enzymatic cleavage, and activation or destruction by proteolysis, remain largely unknown.

### Classification
The Baltimore classification, developed by <b>Dr. David Baltimore</b>, is a virus classification system that groups viruses into families, depending on their type of genome (DNA, RNA, single-stranded (ss), double-stranded (ds), etc.) and their method of replication.

Classifying viruses according to their genome means that those in a given category will all behave in much the same way, which offers some indication of how to proceed with further research.
- <b>I</b>: dsDNA viruses (e.g. Adenoviruses, Herpesviruses, and Poxviruses),
- <b>II</b>: ssDNA viruses (+ strand or "sense") DNA (e.g. Parvoviruses),
- <b>III</b>: dsRNA viruses (e.g. Reoviruses),
- <b>IV</b>: (+)ssRNA viruses (+ strand or sense) RNA (e.g. Picornaviruses, and Togaviruses),
- <b>V</b>: (−)ssRNA viruses (− strand or antisense) RNA (e.g. Orthomyxoviruses, and Rhabdoviruses),
- <b>VI</b>: ssRNA-RT viruses (+ strand or sense) RNA with DNA intermediate in life-cycle (e.g. Retroviruses), and
- <b>VII</b>: dsDNA-RT viruses (e.g. Hepadnaviruses).

<img src="assets/images/baltimore.classification.gif" align="center" alt="Baltimore Classification">

Visualization of the <i>7 groups of virus</i> according to the <b>Baltimore Classification</b>.
